bam_masterdata/cli/cli.py

```diff
@@ -658,11 +664,20 @@ def run_parser(
             props=obj_props,
         )
         object_openbis.save()
+
         # save local and openbis IDs to map parent-child relationships
         openbis_id_map[object_id] = object_openbis.identifier
         click.echo(
             f"Object {obj_props.get('$name')} stored in openBIS collection {collection_name}."
         )
+    for _, files in files_parser.items():
+        # Upload the file to openBIS
+        dataset = collection_openbis.new_dataset(
```

Comment:
just for me to confirm: can you directly attach a list of files as `files` in new_dataset? Looks like right, but just want to be 100% you quickly tested this locally yourself.